documentation/4.0/content/release-notes.md

```diff
@@ -451,4 +451,4 @@ Updated several dependencies, including the Oracle Linux base for the container
 
 | Issue | Description |
 | --- | --- |
-| None currently |  |
+| Deadlock on WebLogic Managed Coherence Server startup with Oracle Coherence 12.2.1.3.20. | Intermittently, a deadlock may occur during deployment of the `CoherenceModule` which prevents a WebLogic Managed Coherence Server from reaching `RUNNING` state.  This issue has been resolved in Oracle Coherence versions 12.2.1.4.0 and above. |
```

Comment:
12.2.1.3.20 -> ??? Is this supposed to be 12.2.1.3.**20** or 12.2.1.3.2.0
`CoherenceModule` which -> `CoherenceModule`, (comma) which
reaching `RUNNING` state -> reaching the `RUNNING` state
and above. -> and later.

Reply:
This version is logged by Coherence so it's 12.2.1.3.20.

Reply:
Oh, ok. I've never seen it before but trust that you know it's correct. Thanks